apps/greencheck/tests/test_importer_csv.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+import pathlib
+import re
+import pandas as pd
+import ipdb
+from io import StringIO
+
+from django.core.management import call_command
+from apps.greencheck.importers.importer_csv import CsvImporter
+
+from django.conf import settings
+
+
+@pytest.fixture
+def sample_data_raw():
+    """
+    Retrieve a locally saved sample of the population to use for this test
+    Return: CSV
+    """
+    this_file = pathlib.Path(__file__)
+    csv_path = this_file.parent.parent.joinpath("fixtures", "test_dataset_csv.csv")
```

Comment:
You're already using pathlib correctly here. Danke!

This should be a good reference for the implementation above.